iocore/net/quic/QUICHandshake.h

```diff
@@ -50,6 +50,7 @@ class SSLNextProtocolSet;
 class QUICHandshake : public QUICApplication
 {
 public:
+  QUICHandshake(QUICConnection *qc, SSL_CTX *ssl_ctx, INK_MD5 token);
```

Comment:
Should we have the both constructors? Maybe you can use default arguments for `token`.
http://en.cppreference.com/w/cpp/language/default_arguments

Reply:
I'm fine with having the two constructors but I don't want to have exactly the same code in here and there.

Reply:
I'd prefer to removing the original `QUICHandshake(QUICConnection *qc, SSL_CTX *ssl_ctx);` since the token should never be same with each other, and the server must send a statless retry token.

Reply:
That sounds reasonable.